benchmarks/setup/BestPracticesWindows.md

```diff
@@ -0,0 +1,16 @@
+# Best Practices Windows
+
+Running the benchmarks is possible with the help of the Windows Subsystem for Linux.
```

Comment:
Could we reference here how to setup a subsystem for linux? Any resources do you used?